pyteal/compiler/constants_test.py

```diff
@@ -63,6 +64,31 @@ def test_extractAddrValue():
         assert actual == expected
 
 
+def test_extractMethodValue():
+    tests = [
+        (TealOp(None, Op.method, "create(uint64)uint64"), b"\x43\x46\x41\x01"),
+        (TealOp(None, Op.method, "update()void"), b"\xa0\xe8\x18\x72"),
+        (TealOp(None, Op.method, "optIn(string)string"), b"\xcf\xa6\x8e\x36"),
+        (TealOp(None, Op.method, "closeOut()string"), b"\xa9\xf4\x2b\x3d"),
+        (TealOp(None, Op.method, "delete()void"), b"\x24\x37\x8d\x3c"),
+        (TealOp(None, Op.method, "add(uint64,uint64)uint64"), b"\xfe\x6b\xdf\x69"),
+        (TealOp(None, Op.method, "empty()void"), b"\xa8\x8c\x26\xa5"),
+        (TealOp(None, Op.method, "payment(pay,uint64)bool"), b"\x3e\x3b\x3d\x28"),
+        (
+            TealOp(
+                None,
+                Op.method,
+                "referenceTest(account,application,account,asset,account,asset,asset,application,application)uint8[9]",
+            ),
+            b"\x0d\xf0\x05\x0f",
+        ),
```

Comment:
These tests should have quotes around the method signature args to be accurate

Reply:
This is taken from your test script: https://gist.github.com/jasonpaulos/99e4f8a75f2fc2ec9b8073c064530359, and the tests come from hash without quotes...